Yona/Yona/Others/SMSValidPasscodeLogin/LoginSignupValidationMasterView.swift

```diff
@@ -77,29 +76,26 @@ extension LoginSignupValidationMasterView {
 //MARK: - Button logic code used on 2 screens
 extension LoginSignupValidationMasterView {
     func pinResetTapped() {
-        if NSUserDefaults.standardUserDefaults().boolForKey(YonaConstants.nsUserDefaultsKeys.isBlocked) {
-            
-            PinResetRequestManager.sharedInstance.pinResetRequest({ (success, pincode, message, code) in
-                if success {
-                    print(pincode!)
-                    if pincode != nil {
-                        
-                        let timeToDisplay = pincode!.convertFromISO8601Duration()
-                        setViewControllerToDisplay("SMSValidation", key: YonaConstants.nsUserDefaultsKeys.screenToDisplay)
-                        let localizedString = NSLocalizedString("login.user.pinResetReuestAlert", comment: "")
-                        let alert = NSString(format: localizedString, timeToDisplay!)
-                        self.displayAlertMessage("", alertDescription: String(alert))
-                        
-                        if let sMSValidation = R.storyboard.sMSValidation.sMSValidationViewController {
-                            self.navigationController?.pushViewController(sMSValidation, animated: false)
-                        }
+        PinResetRequestManager.sharedInstance.pinResetRequest({ (success, pincode, message, code) in
+            if success {
+                print(pincode!)
+                if pincode != nil {
+                    
+                    let timeToDisplay = pincode!.convertFromISO8601Duration()
+                    setViewControllerToDisplay("SMSValidation", key: YonaConstants.nsUserDefaultsKeys.screenToDisplay)
+                    let localizedString = NSLocalizedString("login.user.pinResetReuestAlert", comment: "")
+                    let alert = NSString(format: localizedString, timeToDisplay!)
+                    self.displayAlertMessage("", alertDescription: String(alert))
+                    
+                    if let sMSValidation = R.storyboard.sMSValidation.sMSValidationViewController {
+                        self.navigationController?.pushViewController(sMSValidation, animated: false)
                     }
-                } else {
-                    //TODO: Will change this after this build
-                    self.displayAlertMessage("Error", alertDescription: "User not found")
                 }
-            })
-        }
+            } else {
+                //TODO: Will change this after this build
+                self.displayAlertMessage("Error", alertDescription: "User not found")
```

Comment:
@chandanvarma009 "Error" & "User not found" - use from localization file rather to add here.